src/pages/api/users/[id].js

```diff
@@ -0,0 +1,30 @@
+import dbConnect from "@/utils/dbConnect";
+import { deleteUser, modifyUser } from "@/utils/dbFunctions";
+
+export default async function handler(req, res) {
+  const { method } = req;
+
+  await dbConnect();
```

Comment:
Do we need to close the connection? 🤔 
when we are done with the request I mean.

Reply:
Good question! No, it's not necessary. It can be done easily with .close() but the recommended way is to keep it open unless you have a specific reason to close it.